### PR TITLE
Remove Go module cache from sdk image to shrink size

### PR DESCRIPTION
**Pull request by jdolitsky**

### CODE REVIEW

**Review by kaniini — APPROVED** [review]

File: images/sdk/configs/latest.melange.yaml [hunk 1]
```diff
@@ -58,13 +58,9 @@ pipeline:
       git clone https://github.com/chainguard-dev/apko.git
       (cd apko && make apko install)
 
+      # move all Go-built stuff into /usr/bin, remove uneeded files
+      mv "${GOPATH}/bin/*" "${DESTDIR}/usr/bin"
```
Comment (kaniini) [inline images/sdk/configs/latest.melange.yaml:62]: I don't think we need this part

Comment (jdolitsky) [reply images/sdk/configs/latest.melange.yaml:62]: just pushed up a new version. trying to get goimports into the package

Comment (kaniini) [reply images/sdk/configs/latest.melange.yaml:62]: Why?  It's not a hard requirement for building, and I don't think people really use this image to build apko and melange itself, but rather to make use of them.

Comment (jdolitsky) [reply images/sdk/configs/latest.melange.yaml]: > I don't think people really use this image to build apko and melange itself

i do personally :)

Comment (kaniini) [reply images/sdk/configs/latest.melange.yaml]: Carry on, then :)

**Pull request merged**